src/services/message-service.ts

```diff
@@ -35,6 +35,24 @@ export const createEnterQueueMessage = (
   return createMessage(msg, 'ENTER_QUEUE');
 };
 
+export const createEnterMestringQueueMessage = (
```

Comment:
I den andre PRen som ligger ute atm så har jeg skrevet om message builders til klasser med builder pattern, sånn at vi slipper konstruktører som dette. Kan være greit å pulle inn det (branch cleanup chat) vi pusher dette videre.

Vi kan jo også generere denne på samme måte som med IEnterQueueMessage. Ingen grunn som jeg ser til å lage en ny funksjon for dette